homeassistant/components/tts/__init__.py

```diff
@@ -198,6 +203,34 @@ async def async_say_handle(service: ServiceCall) -> None:
             language = service.data.get(ATTR_LANGUAGE)
             options = service.data.get(ATTR_OPTIONS)
 
+            if tts.base_url is None or tts.base_url == get_url(hass):
+                tts.process_options(p_type, language, options)
```

Comment:
Why don't we use the return values of `process_options`?

Reply:
It's only here used for validation. The output is used when the data is actually rendered to an audio file. That is now not done inside this service but will be called when the media source is being resolved.